### PR TITLE
2.x nav

### PR DESCRIPTION
**Pull request by Elvismutinda**

### CODE REVIEW

File: client/src/assets/scss/_navbar.scss [hunk 1]
```diff
@@ -299,12 +315,21 @@
     color: #333;
     box-sizing: border-box;
     display: flex;
-    height: 90px;
+    height: 100%;
```
Comment (nyatindopatrick) [inline client/src/assets/scss/_navbar.scss:318]: This should remain as 90px

File: client/src/assets/scss/_navbar.scss [hunk 2]
```diff
@@ -12,7 +12,7 @@
   border: none;
   border-top: 3px solid #020763;
   box-shadow: 0 3px 0 #dedede;
-  height: 33px;
+  height: 100px;
```
Comment (nyatindopatrick) [inline client/src/assets/scss/_navbar.scss:15]: 100 is too much. around 60 should be fine

File: client/src/assets/scss/_navbar.scss (hunk 1 above)
Comment (Elvismutinda) [reply client/src/assets/scss/_navbar.scss:318]: Noted

File: client/src/assets/scss/_navbar.scss (hunk 2 above)
Comment (Elvismutinda) [reply client/src/assets/scss/_navbar.scss:15]: Noted

**Pull request merged**